internal/security/proxy/config/config.go

```diff
@@ -130,11 +131,18 @@ func (c *ConfigurationStruct) UpdateWritableFromRaw(rawWritable interface{}) boo
 func (c *ConfigurationStruct) GetBootstrap() bootstrapConfig.BootstrapConfiguration {
 	//To keep config file for proxy unchanged in Geneva we need to create a temporary SecretStore struct so that bootstrapHandler can use it to create a secret client
 	//The config file may be changed in the future version and SecretStore can be used directly like other core services
+
+	// Note - see issue #2873 for details on why this default value is set here
+	scheme := c.SecretService.Scheme
+	if scheme == "" {
+		scheme = "https"
```

Comment:
Now that this is going in after Hanoi, do we want the default to be `http`?

Reply:
I believe we discussed in the planning meeting that there is generally no broad support for defaults in code.  I think we should take it out, for the reason stated above.